src/main/java/decodes/tsdb/CpCompDependsUpdater.java

```diff
@@ -671,21 +672,22 @@ private boolean tsDeleted(DbKey tsKey)
 				
 				// If this was an explicit member of a group, remove the TSDB_GROUP_MEMBER_TS record
 				if (wasMember)
+				{
 					for(TimeSeriesIdentifier tsid : grp.getTsMemberList())
 					{
 						if (tsid.getKey() == tsKey)
 						{
 							grp.getTsMemberList().remove(tsid);
 							q = "delete from TSDB_GROUP_MEMBER_TS where "
-								+ "GROUP_ID = " + grp.getGroupId()
+								+ "GROUP_ID = ?"
 								+ " and "
 								+ (theDb.getTsdbVersion() >= TsdbDatabaseVersion.VERSION_9 ? "ts_id" : "data_id")
-								+ " = " + tsKey;
+								+ " = ?";
 							try
 							{
-								tsGroupDAO.doModify(q);
+								dao.doModify(q,grp.getGroupId(), tsKey);
 							}
-							catch (DbIoException ex)
+							catch (SQLException ex)
```

Comment:
Probably need to print error with the specific keys as well.

Reply:
oh, yeah, now that it's just in the query string.

Reply:
Also odd we're replacing system.err.print above, but not here?

Reply:
That was probably just me being lazy, but I'll go through and replace all the instances within this file.